src/StripeTests/Functional/TelemetryTest.cs

```diff
@@ -53,12 +53,12 @@ public void TelemetryEnabled()
         {
             this.ResetStripeClient();
             var fakeServer = FakeServer.ForMockHandler(this.MockHttpClientFixture.MockHandler);
-            fakeServer.DelayMilliseconds = 10;
+            fakeServer.Delay = TimeSpan.FromMilliseconds(20);
```

Comment:
This test was flaky and randomly failed. I think it's because the resolution of the timer used internally by `Task.Delay` is only around ~15 milliseconds, so even when the fake server was supposed to sleep for 10 milliseconds, we sometimes measured a duration of 8-9 milliseconds and the call to `Verify` would fail because we asserted that `d >= 10`.

I've relaxed the test by sleeping for a bit longer (20 milliseconds) and asserting that the duration is >= 15.